cmd/vic-machine/upgrade/upgrade.go

```diff
@@ -64,6 +64,11 @@ func (u *Upgrade) Flags() []cli.Flag {
 			Usage:       "Roll back VCH version to before the previous upgrade",
 			Destination: &u.Rollback,
 		},
+		cli.BoolFlag{
+			Name:        "resetInProgressFlag",
+			Usage:       "Reset the UpgradeInProgress flag",
```

Comment:
feels like here should have a warning message as well. to make sure user does not reset flag during other process is actually running